src/bindings/python/tests/test_frontend/test_frontendmanager.py

```diff
@@ -24,14 +24,15 @@
         clear_place_stat,
     )
 except Exception:
-    print("No mock frontend available")
```

Comment:
can we add a message what users need to set `OV_FRONTEND_PATH`?
am I right that after build the tests pass out of box? but the issues only after we install OV and tests?

Reply:
Yes will, but python imports requires correct set `LD_LIBRARY_PATH`.
The `OV_FRONTEND_PATH` is required later in e.g
```python
fe = fem.load_by_framework(framework=MOCK_PY_FRONTEND_NAME)
```
when frontend manager tries open library.